packages/web-components/src/components/cbp-tag/cbp-tag.scss

```diff
@@ -22,16 +34,25 @@ cbp-tag {
   
   &[color=warning] {
     --cbp-tag-color: var(--cbp-color-text-darker);
+    --cbp-tag-color-dark: var(--cbp-color-info-darker);
+
     --cbp-tag-color-background: var(--cbp-color-warning-base);
+    --cbp-tag-color-background-dark: var(--cbp-color-warning-light);
   }
 
   &[color=success] {
     --cbp-tag-color: var(--cbp-color-success-lighter);
-    --cbp-tag-color-background: var(--cbp-color-success-dark);
+    --cbp-tag-color-dark: var(-cbp-color-success-darker);
+
+    --cbp-tag-color-background: var(--cbp-color-info-dark);
+    --cbp-tag-color-background-dark: var(--cbp-color-success-light);
   }
 
   &[color=danger] {
     --cbp-tag-color: var(--cbp-color-text-lightest);
+    --cbp-tag-color-dark: var(-cbp-color-text-darkest);
```

Comment:
I could tell this color was wrong by eyeballing it, but I just noticed you're missing a `-` on your variable reference, which is why it's broken.